services/mailer/notify.go

```diff
@@ -208,8 +208,5 @@ func (m *mailNotifier) RepoPendingTransfer(ctx context.Context, doer, newOwner *
 }
 
 func (m *mailNotifier) WorkflowRunStatusUpdate(ctx context.Context, repo *repo_model.Repository, sender *user_model.User, run *actions_model.ActionRun) {
-	if !run.Status.IsDone() {
-		return
-	}
 	MailActionsTrigger(ctx, sender, repo, run)
```

Comment:
The last one from me, then the function `MailActionsTrigger` is unnecessary now, all the code could be extract into this function.